compose/bin/download

```diff
@@ -21,6 +21,16 @@ bin/setup-composer-auth
 
 bin/fixowns
 
-bin/clinotty composer create-project --repository=https://repo.magento.com/ magento/project-"${EDITION}"-edition="${VERSION}" .
+bin/clinotty composer --no-install create-project --repository=https://repo.magento.com/ magento/project-"${EDITION}"-edition="${VERSION}" .
+
+# If Magento in version >= 2.3.7 and < 2.4.4
+if (( $(echo "$VERSION_ROOT < 244" | bc -l) )) && (( $(echo "$VERSION_ROOT >= 237" | bc -l) )); then
+  echo "Adding Magento modules to Composer allow-plugins directive..."
+  composer config allow-plugins.dealerdirect/phpcodesniffer-composer-installer true
```

Comment:
I wonder why the lines below use `bin/clinotty composer ...` and this one doesn't?